widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/ResourceExtensions.kt

```diff
@@ -16,3 +16,7 @@ fun Context.getDimenRes(@DimenRes dimenId: Int): Float {
 fun Context.getDimenResPx(@DimenRes dimenId: Int): Int {
     return this.resources.getDimensionPixelSize(dimenId)
 }
+
+fun String.separateStringWithSymbol(symbol: String): String {
+    return this.split("").joinToString(symbol)
+}
```

Comment:
NIT: You can ignore this but I think it might be better to extract it to some String dedicated utils class or a dedicated String extensions class.

Reply:
Made a PrimitiveExtensions.kt class, might rename it later because technically String is not a primitive, but I would like to have String, Int, Long etc extensions in one place